apps/examples/src/index.tsx

```diff
@@ -1,4 +1,10 @@
-import { DefaultErrorFallback, ErrorBoundary } from '@tldraw/tldraw'
+import { getBundlerAssetUrls } from '@tldraw/assets/src/urls'
```

Comment:
This points to a .ts file which will be problematic for a lot of folks.

I wonder whether we could write the assets package with .js files and JSDoc typings to avoid that.

Reply:
ohhhh that might be a good idea actually. this is more for us internally for now, but I'll look into that in a follow-up!